src/NuGet.Clients/PackageManagement.VisualStudio/ProjectSystems/BuildIntegratedProjectSystem.cs

```diff
@@ -62,6 +64,30 @@ private IScriptExecutor ScriptExecutor
             }
         }
 
+        public IVsProjectBuildSystem ProjectBuildSystem
+        {
+            get
+            {
+                if (!_buildSystemFetched)
```

Comment:
Agreed. Changed. Also changed in VSMSBuildNuGetProjectSystem (I made my code consistent with this). Thanks.